api/app/views/notification_mailer/comment_notification.html.erb

```diff
@@ -0,0 +1,15 @@
+<%
+  if @comment.subject.is_a? Resource
+    title = @comment.subject.title.titleize
```

Comment:
I'm seeing a lot of Demeter violations in these mailer views that could lead to problems. I'm also seeing a lot of logic in them that should probably be elsewhere. Maybe touch based with @scryptmouse and see if she has suggestions for moving logic out of the views and addressing these Demeter violations.

Reply:
For things like this, I suggest we add `draper` so we can be using decorators in the mailer views. Most of our frontend is handled by serializers, but I think for the mailer views, the separation between models and their presentational concerns that it offers is 100% better than complex view logic.

This logic could easily end up in a `CommentDecorator` that is encapsulated and easy to test with its individual methods. That could also be where the URL generation winds up, getting it out of the model but still easily accessible (with draper, any model can be decorated simply with `model.decorate`). Talk to me before implementing if we agree we should go this route, as I can offer some pointers.

Also, as a general rule, branching via `is_a?` / `kind_of?` should never end up in a view. That sort of logic should be in the model / decorator

Reply:
I'm ok with using draper here.